src/gpodder/gtkui/main.py

```diff
@@ -2210,7 +2207,6 @@ def play_or_download(self, current_page=None):
             return (False, False, False, False, False)
 
         (can_play, can_download, can_cancel, can_delete) = (False,) * 4
-        (is_played, is_locked) = (False,) * 2
```

Comment:
Removing these unused variables is unrelated to the fix and should happen in a separate PR.

Reply:
I've split this off into https://github.com/gpodder/gpodder/pull/1158